Example/CryptoOnramp Example/CryptoOnramp Example/OAuthScopeSelectionView.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  OAuthScopeSelectionView.swift
+//  CryptoOnramp Example
+//
+//  Created by Michael Liberatore on 10/20/25.
+//
+
+import SwiftUI
+
+/// A view that allows selection of OAuth scopes for authentication.
+struct OAuthScopeSelectionView: View {
```

Comment:
This view was moved to its own file along with minimal UI changes for its new home in a sheet. See the screenshots in the description.